### ISSUE TITLE
IP Address is not shown in MQTT Broker on Home Assistant

### ISSUE THREAD

**Issue by jiwanus** [seaky/nspanel_pro_tools_apk#108]
In some cases the IP address stays as "unavailable" even though sometimes it is passed to mqtt.  MQTT log below:

"device": {
      "id": "0725309b080003xxxxxxxx376d1d868",
      "name": "NSP-Garaz",
      "name_by_user": null,
      "disabled": false,
      "disabled_by": null,
      "entities": [
        {
          "device_class": null,
          "disabled_by": null,
          "disabled": false,
          "entity_category": "diagnostic",
          "entity_id": "sensor.nsp_garaz_ip_address",
          "icon": null,
          "original_device_class": null,
          "original_icon": null,
          "state": {
            "entity_id": "sensor.nsp_garaz_ip_address",
            "state": "unavailable",
            "attributes": {
              "friendly_name": "NSP-Garaz IP Address"
            },
            "last_changed": "2024-06-25T08:51:21.497535+00:00",
            "last_reported": "2024-06-25T08:51:21.497535+00:00",
            "last_updated": "2024-06-25T08:51:21.497535+00:00"
          },
          "unit_of_measurement": null
        },

Checked on FW version 1.7 and 3.0. apk release is 2.2.1

**Comment by seaky** [comment]
Yes i also confirm that I realised this behaviour even though the device reports the address, the HA still shows unavailable.
Currently the device daily reports the IP address maybe I should increase the frequency to 4 times a day.

**Comment by nsleigh** [comment]
I have 2.3.1 installed and am still seeing the IP address as Unavailable (and the other 2 diagnostics) - looking at the history for the last month I can see it is reported occasionally. Is this something I need to configure?
